import-scripts/annotation_suite/annotation_suite_wrapper.sh

```diff
@@ -0,0 +1,128 @@
+    echo -e "\tCENTER_NAME=${CENTER_NAME}"
+    shift
+    ;;
+    -s=*|--sequence-source=*)
+    SEQUENCE_SOURCE="${i#*=}"
+    echo -e "\tSEQUENCE_SOURCE=${SEQUENCE_SOURCE}"
+    shift
+    ;;
+    -p=*|--annotation-suite-scripts-dir=*)
+    ANNOTATION_SUITE_SCRIPTS_HOME="${i#*=}"
+    echo -e "\tANNOTATION_SUITE_SCRIPTS_HOME=${ANNOTATION_SUITE_SCRIPTS_HOME}"
+    shift
+    ;;
+    *)
+    ;;
+esac
+done
+
+if [[ -z ${INPUT_DATA_DIRECTORY} || -z ${OUTPUT_DATA_DIRECTORY} || -z ${MERGED_MUTATION_FILENAME} || -z ${CENTER_NAME} || -z ${SEQUENCE_SOURCE} || -z ${ANNOTATION_SUITE_SCRIPTS_HOME} ]]; then
+    usage
+    exit 1
+fi
+
+if [ -z ${JAVA_SSL_ARGS} ] ; then
+    echo "Cannot run the annotator without SSL args currently. exiting..."
+    exit 2
+fi
+
+PROCESSED_SUB_DIR_NAME="${OUTPUT_DATA_DIRECTORY}/processed"
+ANNOTATED_SUB_DIR_NAME="${OUTPUT_DATA_DIRECTORY}/annotated"
+FILE_EXTENSIONS_LIST="vcf,maf,txt" # text files are treated as MAFs to handle names like data_mutations_extended.txt
+
+STANDARDIZE_MUTATION_DATA_SCRIPT=${ANNOTATION_SUITE_SCRIPTS_HOME}/standardize_mutation_data.py
+GENOME_NEXUS_ANNOTATOR_JAR=${ANNOTATION_SUITE_SCRIPTS_HOME}/annotator.jar
+MERGE_MAFS_SCRIPT=${ANNOTATION_SUITE_SCRIPTS_HOME}/merge_mafs.py
+
+GENOME_NEXUS_ANNOTATOR_ISOFORM="uniprot"
+GENOME_NEXUS_ANNOTATOR_POST_SIZE=1000
+
+function initAndCleanWorkingDirectory {
+    # (1): parent directory containing raw/unprocessed data files
+    data_dir=$1
+    if [ -d $data_dir ] ; then
+        rm -f $data_dir/*
+    else
+        mkdir $data_dir
+    fi
+}
+
+function standardizeMutationFilesFromDirectory {
+    # processed files will be written to ${OUTPUT_DATA_DIRECTORY}/processed
+    echo -e "\t[INFO] standardizeMutationFilesFromDirectory(), standardized mutation files from ${INPUT_DATA_DIRECTORY} will be written to ${PROCESSED_SUB_DIR_NAME}"
+    python3 ${STANDARDIZE_MUTATION_DATA_SCRIPT} --input-directory ${INPUT_DATA_DIRECTORY} --output-directory ${PROCESSED_SUB_DIR_NAME} --center ${CENTER_NAME} --sequence-source ${SEQUENCE_SOURCE} --extensions ${FILE_EXTENSIONS_LIST}
+    if [ $? -gt 0 ] ; then
+        echo -e "\n[ERROR] standardizeMutationFilesFromDirectory(), error encountered while running ${STANDARDIZE_MUTATION_DATA_SCRIPT}"
+        exit 1
+    fi
+}
+
+function annotateMAF {
+    # (1): input file to annotate
+    input_file=$1
+    output_file=${ANNOTATED_SUB_DIR_NAME}/$(basename ${input_file}).annotated
+    echo -e "\t[INFO] annotateMAF(), annotating MAF: ${input_file} --> ${output_file}"
+    java ${JAVA_SSL_ARGS} -jar ${GENOME_NEXUS_ANNOTATOR_JAR} --filename ${input_file} --output-filename ${output_file} --isoform-override ${GENOME_NEXUS_ANNOTATOR_ISOFORM} -r
+    if [ $? -gt 0 ] ; then
+        echo -e "\n[ERROR] annotateMAF(), error encountered while running the genome nexus annotation pipeline"
+        exit 1
+    fi
+}
+
+function annotateStandardizedMAFs {
+    for f in ${PROCESSED_SUB_DIR_NAME}/* ; do
+        if [ -f ${f} ] ; then
```

Comment:
if we allow spaces in filenames, these functions (this one, the one above, and below) need quoting around filenames

Reply:
I think I've put quotes around everything now (will be in next commit)